includes/class-sensei-question.php

```diff
@@ -57,11 +57,8 @@ public function question_types() {
 		 *
 		 * @hook sensei_question_types
 		 *
-		 * @param {array} $types {
```

Comment:
Lack of support for this kind of array detail does seem to be a pretty major limitation of this doc block syntax. We lose quite valuable information.

Reply:
I just tested this and this seems to work:
```
		/**
		 * Filter the question types.
		 *
		 * @hook sensei_question_types
		 *
		 * @param {Array[]} $types        Question types.
		 * @param {string}  $types[].id   Question type ID.
		 * @param {string}  $types[].text Question type label.
		 *
		 * @return {string[]} Associative array of question types.
		 */
```